tutorials/xr/xr_action_map.rst

```diff
@@ -0,0 +1,246 @@
+=================
+
+Godot has an action map feature as part of the XR system. At this point in time this system is part of the OpenXR module but there are plans to encompass WebXR into this in the near future hence we call it the XR action map system in this document. It implements the build in action map system of OpenXR mostly exactly as it is offered.
+
+The XR action map system exposes input, positional data and output for XR controllers to your game. It does this by exposing named actions that can be tailored to your game and binding these to the actual inputs and outputs on your XR devices.
+
+As the XR action map is currently part of the OpenXR module, OpenXR needs to be enabled in your project settings to expose it:
+
+.. image:: img/openxr_settings.png
+
+
+
+You will then find the XR Action Map interface in the bottom of the screen:
+
+.. image:: img/xr_action_map.webp
+
+
+
+.. note::
+	Godots build in input system has many things in common with the XR action map system. In fact our original idea was to add functionality to the existing input system and expose the data to the OpenXR action map system. We may revisit that idea at some point but as it turns out there were just too many problems to overcome. To name a few:
+
+	 * Godots input system mainly centers around button inputs, XR adds triggers, axis, poses and haptics (output) into the mix. This would greatly complicate the input system with features that won't work for normal controllers or contrast with the current approach. It was felt this would lead to confusion for the majority of Godot users.
+	 * Godots input system works with raw input data that is parsed and triggers emitting actions, this input data is made available to the end user. OpenXR completely hides raw data and does all the parsing for us, we only get access to already parsed action data. This inconsistency is likely to lead to bugs when an unsuspecting user tries to use an XR device as a normal input device.
+	 * Godots input system allows changes to what inputs are bound to actions in runtime, OpenXR does not.
+   * Godots input system is based on device ids which are meaningless in OpenXR.
+
+	This does mean that a game that mixes traditional inputs with XR controllers will have a separation. For most applications either one or the other is used and this is not seen as a problem. In the end, it's a limitation of the system.
+
+The default action map
+======================
+
+Godot will automatically create a default action map if no action map file is found.
+
+.. warning::
+	This default map was designed to help developers port their XR games from Godot 3 to Godot 4. As a result this map essentially binds all known inputs on all controllers supported by default, to actions one on one. This is not a good example of setting up an action map but it does allow a new developer to have a starting point when they want to become familiar with Godot XR without designing a proper action map for their game first.
```

Comment:
No, just that I'm writing this on the assumption that we may not change the default map in time and even if we change it, it will likely still be a default map to give people a quick start only.

I'm still very keen on continueing that discussion, we may want to pick it up in next weeks XR meeting.